docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+they should spend their time reacting to the opportunities that these metrics present.
+For that reason (and many others), Elastic has embraced Google's Core web vitals.
+
+[discrete]
+[[user-experience-core-vitals]]
+==== Core web vitals
+
+https://web.dev/vitals/[Core web vitals] is a recent initiative from Google to introduce a new set of
+metrics that better categorize good and bad sites by quantifying the real-world {user-experience}.
+This is done by looking at three key metrics: loading performance, visual stability, and interactivity:
+
+[role="screenshot"]
+image::images/web-dev-vitals.png[Web dev vitals (image source: https://web.dev/vitals)]
+
+Image source: https://web.dev/vitals/[web.dev/vitals]
+
+Largest contentful paint (LCP)::
+Loading performance. LCP is the timestamp when the main content of a page has likely loaded.
+To users, this is the _perceived_ loading speed of your site.
+To provide a good user experience, Google recommends an LCP of fewer than 2.5 seconds.
+footnote:[Source: https://web.dev/lcp/[web.dev]]
+
+First input delay (FID)::
+Load responsiveness. FID measures the time between a user's first interaction with a page, like a click,
+and when the page can respond to those interactions.
+To provide a good user experience, Google recommends a FID of less than 100 milliseconds.
+footnote:[Source: https://web.dev/fid/[web.dev]]
+
+Cumulative layout shift (CLS)::
+Visual stability. Is content moving around because of async resource loading or dynamic content additions?
+CLS measures these frustrating unexpected layout shifts.
+To provide a good user experience, Google recommends a CLS score of less than `.1`.
+footnote:[Source: https://web.dev/cls/[web.dev]]
+
+TIP: Beginning in 2021, Google will start using core web vitals as part of their ranking algorithm
+and will open up the opportunity for websites to rank in the "top stories"
+position without needing to leverage https://amp.dev/[AMP].
+footnote:[Source: https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html[webmasters.googleblog.com]]
+
+[discrete]
+[[user-experience-distribution]]
+=== Load/view distribution
+
+Operating system, browser family, and geographic location can all have a massive impact on how visitors
+experience your website.
+This data can help you understand when and where your users are visiting from, and can help you
+prioritize optimizations -- for example, prioritizing improvements for the most popular browsers visiting your site.
+
+Don't forget, this data also influences search engine page rankings and placement in top stories for content sites --
+without requiring the use of AMP.
+
+[role="screenshot"]
+image::images/visitor-breakdown.png[User experience visitor breakdown]
+
+[discrete]
+[[user-experience-errors]]
+=== Error breakdown
+
+Variation in users' software and hardware makes it nearly impossible to test for every combination.
```

Comment:
I couldn't tell where you were going with the first couple of sentences. I'd try to get to the point in the first sentence, if you can, then develop it.